python/hsfs/feature.py

```diff
@@ -81,7 +89,14 @@ def from_response_json(cls, json_dict):
         return cls(**json_decamelized)
 
     def is_complex(self):
-        """Returns true if the feature has a complex type."""
+        """Returns true if the feature has a complex type.
+
+        !!! example
+            ```python
+            selected_feature = fg.to_dict()["features"][0]
```

Comment:
Is this the only way to get feature object and check if it's complex or not? If so we can skip it